ci/docker/runtime_functions.sh

```diff
@@ -895,6 +895,25 @@ nightly_test_javascript() {
     make -C /work/mxnet/amalgamation libmxnet_predict.js MIN=1 EMCC=/work/deps/emscripten/emcc
 }
 
+#Tests Model backwards compatibility on MXNet
+nightly_model_backwards_compat_test() {
+    set -ex
+    export PYTHONPATH=./python/
+    ./tests/nightly/model_backwards_compatibility_check/model_backward_compat_checker.sh
+}
+
+#Backfills S3 bucket with models trained on earlier versions of mxnet
+nightly_model_backwards_compat_train() {
+    set -ex
+    export PYTHONPATH=./python/
+    VENV=mbcc_py2_venv
+    virtualenv -p `which python2` $VENV
```

Comment:
If you run inside a virtualenv, you won't have access to any installed packages. I'd recommend not to use virtualenv here because you're running inside docker anyways. This is also the reason for the CI error that states it could not find boto3.

Reply:
Ah I see. Fixed in edd68164526cf591b090322f8f6e0805e17c17ee